modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -47,6 +48,12 @@ protected function get_default_children_elements() {
 					'_title' => __( 'Tab #2', 'elementor' ),
 				],
 			],
+			[
+				'elType' => 'container',
+				'settings' => [
+					'_title' => __( 'Tab #3', 'elementor' ),
+				],
+			],
```

Comment:
I think it should not be part of the PR.

Reply:
@iNewLegend , yes , this is something you missed , so I added it 
I don't think its a bug deal , and it definitely should not block the PR

Reply:
Done